### PR TITLE
Added Draft proposal for contributions and approval process in WG

### PR DESCRIPTION
**Pull request by shilpa-padgaonkar**
Pull request for [#19 ](https://github.com/camaraproject/EdgeCloud/issues/19)

This is a draft to propose a way to contribute and follow an approval process within the working group

### CODE REVIEW

File: documentation/Contributions-and-ApprovalProcess.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+**Branching and Approaval process**
+For the API/APIs where multiple contributors showcase capability overlap, the core contributors in a smaller round can agree on the first feature(API) for contribution. If for eg.  feature-1 (which has an overlap) within the intent group AppLCM is planned to be contributed, an issue gets created to mention the brief details such as agreed API name, file name, contributor list etc. This could be contributed as AppLCM-F1.MD file and can include these overview details.
+
+Assuming we have 3 contributors for the agreed feature, the following sample branches could be created:
+
+* MEC-AppLCM-F1
+* 5GFF-AppLCM-F1
+* EXR-AppLCM-F1
+* CONSENSUS-AppLCM-F1
+![image](./images/EdgeCloudWG-Branches-Approvals.PNG)
```
Comment (gainsley) [inline documentation/Contributions-and-ApprovalProcess.md:20]: Link appears to be broken, I think maybe need to change PNG -> png to match image file name.

**Review by gainsley — APPROVED** [review]
Thanks for adding this doc, looks good.

File: documentation/Contributions-and-ApprovalProcess.md (hunk 1 above)
Comment (shilpa-padgaonkar) [reply documentation/Contributions-and-ApprovalProcess.md]: @gainsley : Thanks for the hint Jon. Broken link is now fixed

File: documentation/Contributions-and-ApprovalProcess.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+We propose creating intent based subgroups for APIs, identified for now as different subdirectories within the EdgeCloud repository. (Later these could be own repositories which offer cleaner separation)
+For eg.
+
+* Edge-discovery
+* Application-LCM
+
+<br>
+**Branching and Approaval process**
+For the API/APIs where multiple contributors showcase capability overlap, the core contributors in a smaller round can agree on the first feature(API) for contribution. If for eg.  feature-1 (which has an overlap) within the intent group AppLCM is planned to be contributed, an issue gets created to mention the brief details such as agreed API name, file name, contributor list etc. This could be contributed as AppLCM-F1.MD file and can include these overview details.
```
Comment (Kevsy) [inline documentation/Contributions-and-ApprovalProcess.md:13]: This should align with the ongoing CAMARA 'fork and pull' proposal so this part may have to be tweaked.

**Review by Kevsy — APPROVED** [review]
I think we need to align with the 'fork and pull' approach being proposed for the entire CAMARA project

**Pull request closed without merging**